plugins/netscaler-k8s.yaml

```
@@ -0,0 +1,41 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: netscaler-k8s
+spec:
+  shortDescription: A Kubernetes plugin for inspecting Ingress Controller and associated NetScaler deployments
```

this will get truncated at 50 chars.
```suggestion
  shortDescription: inspect NetScaler Ingresses
```

better?

---

Fixed in latest commit